javalin/src/test/java/io/javalin/staticfiles/TestStaticFiles.kt

```diff
@@ -391,4 +395,26 @@ class TestStaticFiles {
         assertThat(http.get("/file.javalin").headers.getFirst(Header.CONTENT_TYPE)).contains("application/x-javalin")
         assertThat(http.getBody("/file.javalin")).contains("TESTFILE.javalin")
     }
+
+   @Test
+   fun `cannot transverse paths`() = TestUtil.test(Javalin.create { config ->
+       config.staticFiles.add {
+           it.directory = "/public/subdir"
+           it.location = Location.CLASSPATH
+       }
+   }) { _, http ->
+       assertThat(http.get("/index.html").httpCode()).isEqualTo(OK)
+       assertThat(http.get("/../file").httpCode()).isEqualTo(HttpStatus.BAD_REQUEST)
+       assertThat(http.get("/../file").body).isEqualTo("<h1>Bad Message 400</h1><pre>reason: Bad Request</pre>")
```

Comment:
I think this is actually testing the client

Reply:
I checked that the correct origin was sent through the wire